docs/_config.yml

```diff
@@ -0,0 +1,73 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely edit after that. If you find
+# yourself editing this file very often, consider using Jekyll's data files
+# feature for the data you need to update frequently.
+#
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'bundle exec jekyll serve'. If you change this file, please restart the server process.
+#
+# If you need help with YAML syntax, here are some quick references for you:
+# https://learn-the-web.algonquindesign.ca/topics/markdown-yaml-cheat-sheet/#yaml
+# https://learnxinyminutes.com/docs/yaml/
+#
+# Site settings
+# These are used to personalize your new site. If you look in the HTML files,
+# you will see them accessed via {{ site.title }}, {{ site.email }}, and so on.
+# You can create any custom variable you would like, and they will be accessible
+# in the templates via {{ site.myvariable }}.
+
+title: "Algo Builder"
+name: "Algo Builder"
+baseurl: "/algo-builder" # the subpath of your site, e.g. /blog
+url: "https://scale-it.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+repository: "scale-it.github.io/algo-builder"
```

Comment:
```suggestion
url: "https://scale-it.github.io/algo-builder/" # the base hostname & protocol for your site, e.g. http://example.com
repository: "https://github.com/scale-it/algo-builder"
```

Reply:
- Updating URL as per suggested name (but not sure, as base url is supposed to have /algo-builder suffix)
- Cannot update the repository url (the correct format is `user/repo`). So i think it should be `scale-it/algo-builder` (removing `github.io`). Updating to given suggestion gives:
![image](https://user-images.githubusercontent.com/33264364/112880479-1c819980-90e8-11eb-8375-8374cecb6c38.png)

Reply:
I see - OK, let's use what they suggest.